Makefile

```diff
@@ -61,10 +61,8 @@ build: go-spacemesh get-profiler get-postrs-service
 .PHONY: build
 
 get-libs: get-postrs-lib get-postrs-service
-.PHONY: get-libs
```

Comment:
Why remove this? `get-libs` is not a file name?

Reply:
.PHONY here doesn't do anything.

It is not required for non-filename targets, but for targets that you always want to rerun no matter if their _input_ has changed (or have no _input_). `get-libs` have well-defined input (it is the path to the .zip file) and make can automatically decide to run it if needed.